internal/backup_util_test.go

```diff
@@ -53,3 +55,37 @@ func TestGetBackupTimeSlices_OrderCheck(t *testing.T) {
 	assert.True(t, result[0].BackupName == testStreamBackup.BackupName+".2", "GetBackupTimeSlices returned bad time ordering: Second file should be first, because second was added last")
 	assert.True(t, result[0].Time.After(result[1].Time), "GetBackupTimeSlices returned bad time ordering: order should be Descending")
 }
+
+func TestGetGarbageFromPrefix(t *testing.T) {
+	backupNames := []string{"backup", "garbage", "garbage_0"}
+	folders := make([]storage.Folder, 0)
+	nonGarbage := []internal.BackupTime{{"backup", time.Now(), "some_postfix.json"}}
```

Comment:
Third field of BackupTime is WalFileName, since this is Postgres-specific and not used here, just set is as `ZZZZZZZZZZZZZZZZZZZZZZZZ`. We should eventually get rid of it in future WAL-G versions by extracting the WalFileName to the `postgres` package.

Reply:
Done